src/components/discovery/DiscoveryQueryBuilder.js

```diff
@@ -133,6 +169,7 @@ const DiscoveryQueryBuilder = ({ activeDataset, dataTypeForms, requiredDataTypes
               setFormRef={setFormRef}
               onChange={handleChange}
               handleVariantHiddenFieldChange={handleVariantHiddenFieldChange}
+              conditionValidator={conditionValidator}
```

Comment:
since this is always the same value, I think it should be moved to utils/search instead of being passed as a prop, and then imported for both uses.